ios/RN/RNCamera.m

```diff
@@ -2367,15 +2372,22 @@ - (void)captureOutput:(AVCapturePhotoOutput *)output didFinishProcessingPhoto:(A
     if (error == nil && self.photoTakenResolve != nil) {
         
         NSData *imageData = [photo fileDataRepresentation];
+        
+        UIImage *takenImage = [UIImage imageWithData:imageData];
+        if (self.options && [self.options[@"width"] integerValue]) {
+            takenImage = [RNImageUtils scaleImage:takenImage toWidth:[self.options[@"width"] integerValue]];
```

Comment:
nit: we can update imageData to be  tj_UIImageHEICRepresentation(takenImage, 1) here as well?

Reply:
Yeah, this entire delegate method invoked only for heic. I meant we can move it inside the condition, then we don't need to create `croppedImage` and just to use `imageData`. Just a nit, no action needed:)